components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/store/SessionDataStore.java

```diff
@@ -145,6 +150,8 @@
     private boolean sessionDataCleanupEnabled = true;
     private boolean operationDataCleanupEnabled = false;
     private static boolean tempDataCleanupEnabled = false;
+    private static BinaryJedis jedis;
+    private boolean enableRedis= false;
```

Comment:
Shall we use a more comprehensible variable name? The current name doesn't clearly denote if it is a boolean IMO.